docs/src/gh-pages.md

```diff
@@ -0,0 +1,13 @@
+# Hosting on Github Pages
+
+So you have set up oranda, set up some links and deployed it to Github pages with CI and now the links won't work?
```

Comment:
so, one thing that's worth noting is that not all github pages deployments are not at the root. personal/org gh pages (e.g. axodotdev.github.io) are actually at the root. so it may be useful to identify this particular scenario with the example (your line 7) first instead of assuming they are already at the failure state.

Reply:
fixed